clients/es-integration/src/integration-test/java/com/github/ymikevich/es/integration/services/DefaultTweetServiceIT.java

```diff
@@ -0,0 +1,223 @@
+package com.github.ymikevich.es.integration.services;
+
+import com.github.ymikevich.es.integration.EsIntegrationApplicationContextIT;
+import com.github.ymikevich.es.integration.api.model.Tweet;
+import com.github.ymikevich.es.integration.api.model.TwitterUser;
+import com.github.ymikevich.es.integration.controllers.TweetController;
+import com.github.ymikevich.es.integration.repository.TweetRepository;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = EsIntegrationApplicationContextIT.class)
+@SpringBootTest(classes = EsIntegrationApplicationContextIT.class)
+public class DefaultTweetServiceIT {
+
+    @Autowired
+    WebApplicationContext context;
+
+    @Autowired
+    TweetRepository repository;
+
+    @Autowired
+    TweetController controller;
```

Comment:
if you're writing mvc test - all communications with rest layer should be done via MockMvc instance and not of directly acquiring controllers